package.json

```
@@ -15,7 +15,9 @@
     "build-storybook": "build-storybook",
     "lint-staged": "lint-staged",
     "chromatic": "chromatic --exit-zero-on-changes",
-    "generate": "plop"
+    "generate": "plop",
+    "test:cy": "cypress open",
+    "test": "concurrently 'yarn dev' 'wait-on http://localhost:6006 && yarn test:cy'"
```

```suggestion
    "test": "concurrently \"yarn dev\" \"wait-on http://localhost:6006 && yarn test:cy\""
```

윈도우에서 오류를 뱉는 것 같아서 (윈도우 기준) 되는 형태로 수정해봤는데.. 맥에서도 괜찮으시다면 이 형태로 수정 부탁드려요 ㅎㅎ..

---

제 로컬에서는 문제 없이 돌아가는 것 같아서 package.json 수정할 때 함께 반영하겠습니다. 
고마워요 윈도우맨!!!